src/app/index.tsx

```diff
@@ -1,31 +1,21 @@
 import React from 'react';
-import {
-  SafeAreaView,
-  ScrollView,
-  View,
-  Text,
-  StatusBar,
-  Image
-} from 'react-native';
-
-import bookImage from './assets/img_book1.png';
+import { SafeAreaView, FlatList, StatusBar } from 'react-native';
+import Book from '@components/Book';
 
+import { BOOKS } from './mockupData';
 import styles from './styles';
 
 const App = () => {
   return (
     <>
       <StatusBar barStyle="dark-content" />
       <SafeAreaView style={styles.container}>
-        <ScrollView style={styles.scrollView}>
-          <View style={styles.bookContainer}>
-            <Image source={bookImage} resizeMode="contain" />
-            <View style={styles.decriptionContainer}>
-              <Text style={styles.title}>A Little Bird Told Me</Text>
-              <Text style={styles.author}>Timothy Cross</Text>
-            </View>
-          </View>
-        </ScrollView>
+        <FlatList
+          data={BOOKS}
+          keyExtractor={item => item.id}
+          renderItem={({ item }) => <Book uri={item.image_url} title={item.title} author={item.author} />}
+          style={styles.list}
+        />
```

Comment:
* Extract this FlatList as a new App component => `src/app/components/BookList`
* Inside  the `BookList` component extract the functions  `keyExtractor` and `renderItem` outside the return  function:
```
function BookList() {
...
renderItem = ({item}) => <Book ... />); + TS 
keyExtractor = (item) => `${item.id}`; + TS
...
return (...);
}
```